packages/migration-service/src/gen-transactions.ts

```diff
@@ -12,6 +12,7 @@ import {
   registrarKey,
   PROGRAM_ID as VSR_PROGRAM_ID,
 } from "@helium/voter-stake-registry-sdk";
+import * as Collections from "typescript-collections";
```

Comment:
No preference but https://www.npmjs.com/package/mnemonist looks more maintained?

Reply:
Oh yeah this was just already there in the merkle lib